core/src/tile/labels/labelContainer.cpp

```diff
@@ -34,6 +41,8 @@ bool LabelContainer::addLabel(MapTile& _tile, const std::string& _styleName, Lab
 }
 
 void LabelContainer::updateOcclusions() {
+    m_currentZoom = m_view->getZoom();
```

Comment:
Just figured out it was also needed by the first label update to get the view-projection matrix but was actually not used, fixed by last commit.